src/components/Pagination.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
You can only import  the `range` function from `lodash` using `webpacks` tree shaking instead of importing the entire library like this
```js
import {range} from "lodash";
```

Reply:
your right about that, let me refactor it.thx